packages/cms-lib/functions.js

```diff
@@ -131,6 +131,8 @@ function createFunction(
   if (fs.existsSync(functionFilePath)) {
     logger.error(`The JavaScript file at "${functionFilePath}" already exists`);
     return;
+  } else {
+    fs.ensureFileSync(functionFilePath);
```

Comment:
What is the purpose of this change?

Reply:
So `ensureDir` in `commands/create.js` will check if the directory exists and, if not, create it. This was the one case that relied on the folder being created by that. So when I changed it from `fs.ensureDir` to `fs.exists` this needed to be added to create the folder.